src/metabase/util.clj

```diff
@@ -200,40 +200,42 @@
 
      (date-trunc :month).
      ;; -> #inst \"2015-11-01T00:00:00\""
-  (^java.sql.Timestamp [unit]
+  (^java.sql.Timestamp [unit timezone-id]
    (date-trunc unit (System/currentTimeMillis)))
-  (^java.sql.Timestamp [unit date]
-   (let [trunc-with-format (fn trunc-with-format
+  (^java.sql.Timestamp [unit date timezone-id]
```

Comment:
This is definitely going to break things, there's lots of places in the code where we user this function and they're not passing the extra timezone arg